src/material-experimental/mdc-list/list-base.ts

```diff
@@ -6,19 +6,51 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {AfterContentInit, ElementRef, NgZone, OnDestroy, QueryList} from '@angular/core';
-import {setLines} from '@angular/material/core';
+import {Platform} from '@angular/cdk/platform';
+import {
+  AfterContentInit,
+  Directive,
+  ElementRef,
+  HostBinding,
+  NgZone,
+  OnDestroy,
+  QueryList
+} from '@angular/core';
+import {RippleConfig, RippleRenderer, RippleTarget, setLines} from '@angular/material/core';
 import {Subscription} from 'rxjs';
 import {startWith} from 'rxjs/operators';
 
-export class MatListBase {}
+@Directive()
+export class MatListBase {
```

Comment:
Should this be an abstract class? That way the `_isNonInteractive` can be enforced as well.

Reply:
I actually want that property to default to false, but sure I can make it an abstract class